packages/inter-protocol/src/vaultFactory/prioritizedVaults.js

```diff
@@ -108,9 +115,16 @@ export const makePrioritizedVaults = store => {
     return key;
   };
 
-  const removeVaultsBelow = crKey => {
-    // crKey represents a collateralizationRatio based on the locked price.
+  const removeVaultsBelow = ({ margin, quote, interest }) => {
+    // crKey represents a collateralizationRatio based on the locked price, the
+    // interest charged so far, and the liquidation margin.
     // We'll remove all vaults below that ratio, and return them.
+    const crKey = normalizedCollRatioKey(quote, interest, margin);
+
+    trace(
+      `Liquidating vaults worse than`,
+      normalizedCollRatio(quote, interest, margin),
```

Comment:
this is calling `normalizedCollRatio` even when trace is off, duplicating the work in `normalizedCollRatioKey`.

Reply:
on second thought, not a problem. I first thought you’d just calculate the float then use it in both places, but the `…Key` function does the calculating too. You could separate the encoding but that is complexity is not worth the savings in extra arithmetic.